### PR TITLE
Translation(Japanese) upgrade

### PR DESCRIPTION
**Pull request by yambe2002**
Upgrade Japanese translation.
(1) Modify mistranslations
(2) Update

### CODE REVIEW

File: translations/ja_JP/readme.md [hunk 1]
```diff
@@ -159,7 +159,7 @@
 
     ```javascript
 
-    // if/else/for/while/try は常にスペース、中括弧、複数の行に架かるるようにします
+    // if/else/for/while/try は常にスペースをもち、波括弧が複数の行に架かるるようにします
```
Comment (Bambina) [inline translations/ja_JP/readme.md]: I think "架かるる" should be "架かる", right ? :)

Comment (kachick) [inline translations/ja_JP/readme.md]: > 架かる

:+1:

Comment (yambe2002) [inline translations/ja_JP/readme.md]: Oh thanks. You two are correct.
Line 162 should be:
// if/else/for/while/try は常にスペースをもち、波括弧が複数の行に架かるようにします

Comment (Bambina) [inline translations/ja_JP/readme.md]: My pleasure :bowtie:

**Pull request merged**